### DISCUSSION TITLE
MessageBoxAction does not work in multiplayer

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#13721] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

message box created by MessageBoxAction won't show up in multiplayer

### Reproduction steps

1. create an event that uses MessageBoxAction
2. trigger it in multiplayer
3. message box won't show up

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.3.0.2

### Which operating system did you encounter this bug on?

Windows

**Comment by evilfactory** [comment]
Thank you for the bug report, networking for the MessageBoxAction has now been implemented and will be coming in the DLC Companion Update
